packages/claim-cli/src/utils/network-params.ts

```diff
@@ -0,0 +1,23 @@
+export interface NetworkParams {
+	api: string;
+	rpc: string;
+	l2Claim: string;
+	maxFeePerGas: bigint;
+	maxPriorityFeePerGas: bigint;
+}
+
+export const Mainnet = {
+	api: 'https://token-claim-api.lisk.com/rpc',
+	rpc: 'https://rpc.api.lisk.com',
+	l2Claim: '0xD7BE2Fd98BfD64c1dfCf6c013fC593eF09219994',
+	maxFeePerGas: 1002060n,
+	maxPriorityFeePerGas: 1000000n,
```

Comment:
Isn't it is better to define like this?
```suggestion
	maxPriorityFeePerGas: BigInt(1000000),
```

Reply:
Updated to using `BigInt`